### ISSUE TITLE
Adventuring Fellow is not gaining exp

### ISSUE THREAD

**Issue by trodeghero** [EdenServer/community#4114]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details

/ver 30181205_0

I unlocked the adventuring fellow on an alt and have summoned the npc on 4 different occassions with the purpose to level it up; however, the npc is still level 30 and when I 'talk' to the npc while it's out I get the phrase 'I just gained a level...'. I have posted in discord and have talked with several other players and have not been able to find another with this issue and am unable to determine the reason why they are not gaining exp.

**Comment by KnowOne134** [comment]
assault? Are you fighting, level of mob, level of you, and are solo or in a party?

**Comment by trodeghero** [comment]
Ah I should've been more clear on that.  I fought mobs ranging from level 34-50 each of the times I called it.  These were all done solo and my level was 70+ each of the times summoned.

**Comment by KnowOne134** [comment]
The fellow gains xp as if it were the target level of a party. So it would be the xp as if it was 30 - the party amount. At mobs level 34 it would gain the xp of fighting a vt. So 180 to 200 a kill? And needing x amount of xp to level up

**Comment by trodeghero** [comment]
It should've at least gained 1 level over 4 summons killing that level of mobs, roughly 30+ mobs killed per summon and the fellow still says "I just gained a level...".  I feel it is somehow bugged and not gaining any exp at all

**Comment by KnowOne134** [comment]
Can pop in a gm ticket to get checked out what it's at and see for sure. Still a small sample size. But not even a level. Killed for all 20 mibs per sommon?

**Comment by trodeghero** [comment]
I killed more than 20 mobs per summon each time. The fellow would only leave after the time period passed, never left at 20 kills.  I'll put in for a GM ticket later when I get back on

**Comment by Mozzy4Ever** [comment]
Can verify this is an issue. Your Adventuring Fellow does not gain XP on mobs that con "Too Weak" to the player, even though the mobs may be higher level than the fellow. In my case I was fighting mobs that were 45-55. I killed over 30 of them without the counter (when talking to my fellow) increasing. Also of note is that my bond only supports killing 15, and he was still around. I switched to kill mobs 55+ so they'd give me the most meager amount of XP and my fellow's counter went up. He disappeared after 15 kills of mobs that gave me XP.
